homeassistant/components/fritz/switch.py

```diff
@@ -468,9 +467,9 @@ async def _async_fetch_update(self) -> None:
         self._is_available = True
 
         attributes_dict = {
-            "NewInternalClient": "internalIP",
-            "NewInternalPort": "internalPort",
-            "NewExternalPort": "externalPort",
+            "NewInternalClient": "internal_ip",
```

Comment:
Please adjust all the occurrences of `self._attributes`, so that we use lowercase snake_case consistently. Eg here:
https://github.com/home-assistant/core/blob/e11ffbcdafca67b2d0f4d3059e41851dd9ea6c00/homeassistant/components/fritz/switch.py#L550-L558

Reply:
Thanks. Just did that